python_src/src/lamp_py/tableau/jobs/gtfs_rail.py

```diff
@@ -0,0 +1,400 @@
+import os
+
+import pyarrow
+import pyarrow.parquet as pq
+import sqlalchemy as sa
+
+from lamp_py.tableau.hyper import HyperJob
+
+
+class HyperGTFS(HyperJob):
+    """Base Class for GTFS Hyper Jobs"""
+
+    def __init__(
+        self,
+        gtfs_table_name: str,
+        table_query: str,
+    ) -> None:
+        HyperJob.__init__(
+            self,
+            hyper_file_name=f"LAMP_GTFS_Rail_{gtfs_table_name}.hyper",
+            remote_parquet_path=f"s3://{os.getenv('PUBLIC_BUCKET')}/lamp/tableau/rail/LAMP_GTFS_Rail_{gtfs_table_name}.parquet",
```

Comment:
Open to modifying these default file name and path values, this was just a first attempt

Reply:
i think its worth sorting out all our s3 prefix conventions in the next week ro so.